release.md

```diff
@@ -65,9 +65,10 @@ The publication of the release notes will trigger a Github Actions workflow that
 Make sure this workflow ran without errors - if not, assess why it failed and, if it was a third-party problem (e.g. a network connection issue), try to rerun the workflow.
 However, in the unlikely scenario that the deployment failed, it is more likely that there is something that needs to be fixed, in which case you should make an hotfix right away.
 
+The documentation on Read the Docs will be automatically updated to the new version. To get the latest documentation on the docs portal (docs.pasqal.cloud/pulser), notify the compute team, so that they will manually rebuild and deploy.
```

Comment:
Thanks for adding this!

Two suggestions for the link:
- adding `https://` in front, so that it becomes a real link
- using the `.com` domain (so, `https://docs.pasqal.com/pulser`). Currently it's deployed on two domains, .cloud and .com, but .com is the canonical one. .cloud is legacy and will soon simply redirect to .com.